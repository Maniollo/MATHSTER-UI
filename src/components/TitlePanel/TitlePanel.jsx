import React from 'react'

export const TitlePanel = () =>
    <div className="App-header">
        <h1>MATHSTER</h1>
        <h2>BE THE MATH HERO</h2>
    </div>;