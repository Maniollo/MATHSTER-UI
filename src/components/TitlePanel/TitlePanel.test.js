import React from "react";
import renderer from 'react-test-renderer';
import {TitlePanel} from "./TitlePanel";

test('renders correctly', () => {
    const panel = renderer.create(<TitlePanel/>).toJSON();
    expect(panel).toMatchSnapshot();
})