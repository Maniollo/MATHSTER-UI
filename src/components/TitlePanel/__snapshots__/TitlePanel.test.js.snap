exports[`test renders correctly 1`] = `
<div
  className="App-header">
  <h1>
    MATHSTER
  </h1>
  <h2>
    BE THE MATH HERO
  </h2>
</div>
`;
